data/EDMM/EDNY.toml

```diff
@@ -9,7 +9,7 @@ url = "https://chartfox.org/EDNY"
 [[airport.links]]
 category = "Charts"
 name = "Charts (VFR)"
-url = "https://www.vfraip.de/?EDNY"
+url = "https://aip.dfs.de/BasicVFR/pages/C019CA.html"
 
 [[airport.links]]
 category = "Briefing"
```

Comment:
`https://vats.im/edny-briefing` does not exist as short url.

Reply:
seems like there is no briefing for EDNY provided by EDMM at all, so removed the briefing referral entirely